### DISCUSSION TITLE
Climax bar filling speed

### DISCUSSION THREAD

**Discussion by Derphead** [ErinaSugino/Starbound-SBR-Mod#110] [Trouble Shooting]
I installed a few more mods earlier and one of them, not sure which, makes the climax bar fill much, much slower, to the point npc sexnodes despawn before you can climax (atleast those from Defeat). How do I manually adjust the speed it fills?

**Comment by ErinaSugino** [comment]
If you installed a mod that messes with that, only by editing that specific mod.
